shell/common/switches.h

```diff
@@ -193,6 +193,12 @@ DEF_SWITCH(DisableDartAsserts,
            "disabled. This flag may be specified if the user wishes to run "
            "with assertions disabled in the debug product mode (i.e. with JIT "
            "or DBC).")
+DEF_SWITCH(EnableSerialGC,
+           "enable_serial_gc",
```

Comment:
It looks like the style in this file is to use '-' rather than '_' to separate the words in a flag.

```suggestion
           "enable-serial-gc",
```

Reply:
Done.